web/src/App.css

```diff
@@ -1,40 +1,95 @@
-.App {
+body {
   text-align: center;
-}
+  background-color: #F2F2F2;
+ }
 
-.App-header {
-  background-color: #222;
-  height: 80px;
-  padding: 20px;
-  color: white;
+.App {
+  text-align: left;
+  padding: 100px 0px 50px 100px;
+  background-image: url("https://i.imgur.com/MPcDf2v.jpg");
+  background-repeat: repeat;
+  box-shadow: 0px 1px 5px 1px gray;
 }
 
 .App-title {
-  font-size: 1.5em;
+  font-size: 30px;
+  color: #fff;
+  width: 480px;
 }
 
-.App-intro {
-  font-size: large;
-  padding: 20px 0 0 0;
+.App-instructions {
+  font-size: 12px;
+  color: #fff;
+  padding: 10px 0px 0px 0px;
+  width: 500px;
 }
 
 input[type=text] {
   padding:3px 3px 3px 5px;
   border:1px solid #ccc; 
   border-radius: 4px;
-  box-shadow: 1px 1px 10px #D3D3D3;
 }
 
 input[type=submit] {
-  background: #1990FF;
+  background: #99C432;
   color: #fff;
   margin-left: 5px;
   padding: 3px 5px 3px 5px;
-  border:1px solid #1990FF; 
+  border:1px solid #99C432; 
   border-radius: 4px;
 }
 
-@keyframes App-spin {
-  from { transform: rotate(0deg); }
-  to { transform: rotate(360deg); }
+.User-name {
+  order: 0;
+  flex: 0 1 auto;
+  align-self: auto;
+  text-align: left;
+  font-size: 40px;
+  color: #fff;
+  font-weight: bold;
+  padding: 30px 0px 0px 50px;
+}
+
+.User-update {
+  order: 0;
+  flex: 0 1 auto;
+  align-self: auto;
+  text-align: left;
+  padding: 0px 0px 0px 50px;
+  color: #fff;
+  font-size: 15px;
 }
+
+.User-row {
+  display:flex;
+  flex-direction:row;
```

Comment:
You can consider `flex-wrap: wrap` to have elements jump to next line when there's too little space for them.

Reply:
Sure thing, I add this. :)